cost-optimization-aks-content.md

```diff
@@ -0,0 +1,795 @@
+      [Review small development clusters](###review-small-development-clusters)\
+      [Review your monitoring strategies](###Review your monitoring strategies)\
+      [Review inter-region egress traffic](###Review-inter-region-egress-traffic)
+    :::column-end:::
+    :::column:::
+
+      #### Build Cloud-native applications
+      Prepare your applications to run in such an environment
+      [Understand your application capacity](###understand-your-application-capacity)\
+      [Ensure your application can grow](###ensure-your-application-can-grow)\
+      [Use appropriate requests and limits](###use-appropriate-requests-and-limits)\
+      [Ensure your container is as lean as possible](###ensure-your-container-is-as-lean-as-possible)\
+      [Add pod disruption budget to your application](###add-pod-disruption-budget-to-your-application)\
+      [Set meaningful readiness and liveliness probes](###set-meaningful-readiness-and-liveliness-probes)\
+      [Ensure your container is shutting down gracefully](###ensure-your-container-is-shutting-down-gracefully)\
+      [Consider using retries with exponential backoff](###consider-using-retries-with-exponential-backoff)
+    :::column-end:::
+:::row-end:::
+
+:::row:::
+   :::column:::
+
+      #### Consider Workload Types
+      Optimize cost by configuring your environment according to your workload type
+      [Prepare your environment to fit your workload type](###prepare-your-environment-to-fit-your-workload-type)\
+    :::column-end:::
+   :::column:::
+
+      #### Build a cost-saving Culture
+      Better align developers with your business by designing and iterating for cost optimization early in the development cycle
+      [Spread the cost saving culture](###spread-the-cost-saving-culture)\
+    :::column-end:::  
+:::row-end:::
+
+:::row:::
+    :::column:::
+
+      #### Monitor your spending
+      Monitor your environment and enforce cost-optimized practices
+      [Observe your clusters and watch for recommendations](###observe-your-clusters-and-watch-for-recommendations)\
+      [Monitor costs breakdown using Kubecost](###monitor-costs-breakdown-using-Kubecost)\
+      [Understand how Metrics Server works and monitor it](###understand-how-Metrics-Server-works-and-monitor-it)\
+      [Use Kubernetes Resource Quotas](###use-kubernetes-resource-quotas)\
+      [Enforce cost-optimized practices with Azure Policy](###enforce-cost-optimized-practices-with-azure-policy)\
+      [Design your CI/CD pipeline to enforce cost-saving practices]
+    :::column-end:::
+:::row-end:::
+
+## Optimize your AKS Environment
+Optimize cost with autoscaling, sizing, machine type, etc
+### Optimize AKS Autoscaling
+### Choose the right machine type
+### Select the appropriate region
+### Sign up for Reserved Instances
+### Review small development clusters
+### Review your monitoring strategies
+### Review inter-region egress traffic
+
+## Build Cloud-native applications
```

Comment:
The article leds with multi-tenant.  I think it's perfectly fair to dive down to the responsibilities of a single tenant like you have here.  Because it's the union of all the tenants that cause the user to arrive at the problem statement at the start of this.  Remind the users of this here.